dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -60,6 +60,17 @@ def filter
     render json: {error: e.message}, status: :bad_request
   end
 
+  # GET /api/v1/pd/workshops/upcoming_teachercon
+  def upcoming_teachercon
+    workshops = Pd::Workshop.scheduled_start_on_or_after(Date.today.beginning_of_day).where(
```

Comment:
As per discussion, make sure this only includes the upcoming teachercons associated with this regional partner